assets/js/googlesitekit/api/middleware/preloading.test.js

```diff
@@ -0,0 +1,193 @@
+/**
+ * createPreloadingMiddleware tests.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { addQueryArgs } from '@wordpress/url';
+
+/**
+ * Internal dependencies
+ */
+import createPreloadingMiddleware from './preloading';
+
+const preloadedData = {
+	'google-site-kit/v1/core/site/data/connection': {
+		headers: [],
+		body: {
+			connected: true,
+			resettable: true,
+			setupCompleted: true,
+		},
+	},
+
+	'google-site-kit/v1/core/user/authentication': {
+		headers: [],
+		body: {
```

Comment:
Let's avoid using real endpoints and response data in our tests (where it isn't relevant to the specific test).
The preloading middleware is only slightly different than the core middleware we based it on, and the only detail that is really Site Kit specific is the `timestamp` parameter handling. Apart from that, it will behave the same regardless of what endpoint it is used with.

It also somewhat compels us to maintain/update these if the endpoints or response formats were to change in the future to preserve consistency in the codebase. It also might give the impression to someone less familiar with the codebase that this middleware might only work this way for our endpoints, which isn't the case. Using generic "dummy" data intentionally de-emphasizes bits which are not important (perhaps only for making an assertion) from those that are to better convey intention 🙂 

I think using something like `test/path/a` and `test/path/b` would be just fine 👍